src/anm/analytics.js

```diff
@@ -16,27 +13,15 @@ var Analytics = function () {
             var data = JSON.stringify(self.queue);
             self.queue = [];
 
-            if (supportSendBeacon) {
+            if (navigator.sendBeacon) {
                 navigator.sendBeacon(animatronUrl, data);
             } else {
-                sendViaGif(data);
+                engine.ajax(animatronUrl, null, null, 'POST', null, data, false);
             }
         }
-        if (!supportSendBeacon) {
-            setTimeout(event, timeout);
-        }
     };
 
-    var sendViaGif = function (data) {
-        if (!beacon) {
-            beacon = engine.createStatImg();
```

Comment:
> You made me do this

No way :). An example is still just an example. 

> Anyways, if there's CORS issues then movie can't be loaded at all, since it is loaded via CORS. Am I right?

Dima @skavish should know.